src/System.Data.Common/src/System/Data/Common/DbProviderFactories.cs

```diff
@@ -0,0 +1,199 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+
+namespace System.Data.Common
+{
+    public static partial class DbProviderFactories
```

Comment:
Add xml docs? The .NET Framework ones should be a good starting point.

Reply:
I thought those weren't required? (as they're added elsewhere?) That's why I didn't add them

Reply:
Not sure. @karelz, @saurabh500?

Reply:
Looking at random other classes in this solution, none have xml doc comments.